src/shared-components/Recurrence/CustomRecurrenceModal.tsx

```
@@ -0,0 +1,706 @@
+            </Dropdown>
+          </div>
+
+          {frequency === Frequency.WEEKLY && (
+            <div className="mb-4">
+              <span className="fw-semibold text-secondary">
+                {t('repeatsOn')}
+              </span>
+              <br />
+              <div
+                className="mx-2 mt-3 d-flex gap-1"
+                role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)